tests/bsim/bluetooth/host/l2cap/stress/tests_scripts/l2cap_early_disconnect.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
+# Copyright (c) 2022 Nordic Semiconductor
+# SPDX-License-Identifier: Apache-2.0
+
+source ${ZEPHYR_BASE}/tests/bsim/sh_common.source
+
+# EATT test
```

Comment:
Could you please add test purpose and procedure? Yes, the neighboring tests don't have test description, but it is not good.